core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -107,3 +107,6 @@ def start_board(self, boards: List[Tuple[FlightControllerType, str]]) -> bool:
             raise RuntimeError("Invalid board type: {boards}")
 
         return False
+
+    def restart(self) -> None:
+        self.mavlink_manager.restart()
```

Comment:
Why restart returns `None` if `manvlink_manager.restart` returns a boolean ? The function should also return a boolean no notify users about related problems.

Reply:
Man, I'm now seeing that I selected a wrong stash during the rebase process...
That's why there's old code in there. I had already fixed that :/